compute/is_notifications/notifications/intelligence-stream-significant-impact.adoc

```diff
@@ -2,15 +2,14 @@
 
 The Prisma Cloud https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/vulnerability_management/prisma_cloud_vulnerability_feed[Intelligence Stream(IS)] feed provides information on threat intelligence and vulnerability data, and this is available to all Prisma Cloud Compute Edition and Prisma Cloud Enterprise Edition customers who use the Cloud Workload Protection capabilities. This document captures the fairly rare breaking changes or significant changes to the Intelligence Stream (IS) that impact your deployments and scan results.
 
-The IS is constantly updated and improved to include the latest changes in security vulnerabilities, and new vulnerabilities are frequently added when detected by the Palo Alto Networks threat research teams and official data from third-party feeds such as NVD. All such updates are not captured in this document.
+The IS is constantly updated and improved to include the latest changes in security vulnerabilties, and new vulnerabilities are frequently added when detected by the Palo Alto Networks threat research teams and official data from third party feeds such as NVD. All such updates are not captured in this document.
 
-The items in this list are intended to keep you informed of only the last 7 breaking changes or changes with significant impact, and their items will be rolled out after they have gone into effect.
+The items in this list are intended to keep you informed of only the last 7 breaking changes or changes with significant impact, and it items will be rolled out after they have gone into effect.
 
 [width="90%",cols="1,1,2",options="header"]
 |=========================================================
 |Date Announced |Date Taking Effect |Description of Change
 
-| 22 July, 2022 | 31 July, 2022 | Debian 9 (Stretch) has reached End of Life (EOL), and users of Debian 9 will stop receiving any CVE security vulnerabilities associated with this OS version.
```

Comment:
will you remove this? the change is already live and the  phrasing is different?